Packs/Tidy/Integrations/Tidy/Tidy.py

```diff
@@ -55,7 +54,7 @@ def test(self) -> None:
             ssh.close()
         except AuthenticationException as e:
             raise DemistoException(f"Authentication details isn't valid.\nFull error: {e}")
-        except error as e:
+        except OSError as e:
```

Comment:
Why OSError? Lets do except exception

Reply:
wasnt me, was probably sdk, should i still change?

Reply:
yes